src/utils.js

```diff
@@ -228,4 +228,14 @@ export const hasPropChanged = (oldProps, newProps, key) => {
   return oldProps[key] !== newProps[key];
 };
 
+export const getProjectList = () => {
+  return JSON.parse(localStorage.getItem('project-list') || '{}');
+};
+
+export const saveProject = (id: string, path) => {
+  const list = getProjectList();
+  list[id] = path;
+  localStorage.setItem('project-list', JSON.stringify(list));
+};
```

Comment:
Ah, so localStorage doesn't actually persist between Guppy sessions in the built version. We have to use Electron storage instead.

One maybe-surprising bit of how Guppy projects are organized is that paths to the projects live in a separate reducer, `paths.reducer.js`. The reason for this is that paths are not specific to a project necessarily (the same project might live in different places on different machines).

We already store custom paths with the import-project workflow. I think most of that should be reusable for this.

Essentially, `CREATE_NEW_PROJECT_FINISH` should include the `projectPath` that was stored in `CreateNewProjectWizard` state, and `paths.reducer.js` should listen for this action and add a key for it.

Everything else should work automatically, you shouldn't need either of these utils :)